README.md

```diff
@@ -127,9 +127,20 @@ OpenAISwarm.configure do |config|
 end
 ```
 
-# Examples
+## Quick Start
```

Comment:
https://github.com/graysonchen/ruby-openai-swarm/blob/69344eb8789603268c70ddbbc386b48cc757757d/README.md?plain=1#L19

also, please update it.